imagenet/classify_resnet.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import base64
+import json
+import requests
+from six.moves.urllib.parse import urlparse
+import sys
+import warnings
+
+
+IMAGE_URL = 'https://github.com/spellrun/modelservers/raw/master/imagenet/image.jpg'
+SERVER_URL = 'https://serving.spell.run/marius/resnet/v1/rest/model:predict'
+AUTH_TOKEN = 'thptVN0oyfYM_kz3kWLc-37R6f3m-rs3syWdxBLVNGIR-MjL11DS7nqA89JafAUFKIS9YdZQN8YXDCZnG7VAyTw'
+IMAGE_NET_CLS = 'imagenet1000_clsid_to_human.json'
+
+
+def is_url(path):
+    try:
+        parsed = urlparse(path)
+        return bool(parsed.scheme and parsed.netloc)
+    except ValueError:
+        return False
+
+
+def load_jpeg_img(path):
+    print('Loading JPEG image: {}'.format(path))
+    if is_url(path):
+        res = requests.get(path, stream=True)
+        res.raise_for_status()
+        return res.content
+    else:
+        return open(path, "rb").read()
```

Comment:
nit: ensure the file handle is closed
```suggestion
        with open(path, "rb") as f:
            return f.read()
```